roles/cnf_cert/tasks/teardown.yml

```diff
@@ -1,4 +1,22 @@
 ---
+# Always submit these files to DCI:
+# - Configuration files - tnf_config.yml
+# - Execution logs - dci-tnf-execution.log and cnf-certsuite.log, if exists
+- name: Copy config and log files related to CNF Cert Suite execution
+  ansible.builtin.copy:
+    src: "{{ item }}"
+    dest: "{{ job_logs.path }}"
+    mode: "0750"
+  with_fileglob:
+    - "{{ tnf_dir }}/{{ test_network_function_project_name }}/tnf_config.yml"
+    - "{{ tnf_dir }}/{{ test_network_function_project_name }}/*.log"
+
+# Clean images just after finishing the execution
+- name: Remove local tnf image
+  containers.podman.podman_image:
+    name: "{{ tnf_image }}"
+    state: absent
```

Comment:
what about ignoring errors here to ensure the resource cleanup in the next task?